src/components/Tile.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+// import ourStory from "../assets/our_story.png";
+
+function Tile(props) {
+    return (
+        <section>
+            {props.children}
```

Comment:
Props is hier volgens mij overbodig. Wanneer je in de functie de property children benoemd werkt het wel `function Tile({children}) {`
Let op dat je de children wel tussen **{}** benoemd. Het lijkt nu maar 1 property maar app.js geeft hier meerdere properties aan door (in jou geval meerdere `<p>` elemeneten). Zonder de snorretje werkt het niet.